transformer/dockerfilegenerator/java/gradleanalyser.go

```diff
@@ -450,12 +446,11 @@ func (t *GradleAnalyser) Transform(newArtifacts []transformertypes.Artifact, alr
 				Type: artifacts.JarArtifactType,
 				Configs: map[transformertypes.ConfigType]interface{}{
 					artifacts.JarConfigType: artifacts.JarArtifactConfig{
-						DeploymentFile:                    archiveName,
-						JavaVersion:                       javaVersion,
-						BuildContainerName:                common.DefaultBuildContainerName,
-						DeploymentFileDirInBuildContainer: filepath.Join(t.GradleConfig.AppPathInBuildContainer, buildPath, destinationPath),
-						EnvVariables:                      envVariablesMap,
-						Port:                              common.DefaultServicePort,
+						DeploymentFilePath: archiveName,
```

Comment:
Shouldn't it be filepath.Join(t.GradleConfig.AppPathInBuildContainer, buildPath, destinationPath). Also, we need DeploymentFileName.